src/scripts/assume_role_with_web_identity.sh

```diff
@@ -18,24 +18,29 @@ if [ ! "$(command -v aws)" ]; then
     exit 1
 fi
 
-read -r AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN <<EOF
-$(aws sts assume-role-with-web-identity \
---role-arn "${ORB_EVAL_ROLE_ARN}" \
---role-session-name "${ORB_EVAL_ROLE_SESSION_NAME}" \
---web-identity-token "${CIRCLE_OIDC_TOKEN_V2}" \
---duration-seconds "${ORB_VAL_SESSION_DURATION}" \
---query 'Credentials.[AccessKeyId,SecretAccessKey,SessionToken]' \
---output text)
+for i in {1..3}; do
+    sleep $((RANDOM % 5 + 1))
```

Comment:
I would do this:
```
for i in {0..3}; do
    sleep $((i * 2))
```

That way you don't sleep the first time, and you sleep slightly longer each time through the loop without it being random.